chia/consensus/block_body_validation.py

```diff
@@ -550,28 +547,8 @@ async def validate_block_body(
         if error is not None:
             return error, None
 
-    # create hash_key list for aggsig check
-    pairs_pks: list[G1Element] = []
-    pairs_msgs: list[bytes] = []
-    if conds is not None:
-        pairs_pks, pairs_msgs = pkm_pairs(conds, constants.AGG_SIG_ME_ADDITIONAL_DATA)
-
-    # 22. Verify aggregated signature
-    # TODO: move this to pre_validate_blocks_multiprocessing so we can sync faster
+    # 22. Verify aggregated signature is done in pre-validation
     if not block.transactions_info.aggregated_signature:
         return Err.BAD_AGGREGATE_SIGNATURE, None
```

Comment:
we assert on line 366 that the signatures have been validated. Just a sanity check here that the block had a signature at all. This implements the TODO comment above.